### ISSUE TITLE
AppVeyor worker image update has broken BOINC Windows build

### ISSUE THREAD

**Issue by ChristianBeer** [BOINC/boinc#2178]
Windows CI builds fail with
```
if %TOOLCHAIN_VERSION%==12.0 call "%VS120COMNTOOLS%\vsvars32.bat"
Specify which project or solution file to use because the folder contains more than one project or solution file.
```
since the update yesterday: https://www.appveyor.com/updates/2017/10/22/

Build from yesterday: https://ci.appveyor.com/project/BOINC/boinc/build/1.0.92-pdxxibjc
Build from today: https://ci.appveyor.com/project/BOINC/boinc/build/1.0.92-gltvfenj

**Comment by AenBleidd** [comment]
@ChristianBeer We should switch to newer version of Visual Studio (e.g. 2013)
I can do this if you want

**Comment by AenBleidd** [comment]
Working on this. I post a question to AppVeyor team. Looks like an issue with their scripts

**Comment by JuhaSointusalo** [comment]
Reassigned to @AenBleidd as he is already working on this and actually knows about AppVeyor unlike me.

**Comment by AenBleidd** [comment]
Issue is fixed via #2180

**Comment by AenBleidd** [comment]
@ChristianBeer, I think this ticket can be closed now.
